### PR TITLE
oc adm cordon vs oc adm manage-node node1 --schedulable=false

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-259

### CODE REVIEW

File: modules/nodes-nodes-working-marking.adoc [hunk 1]
```diff
@@ -14,14 +14,16 @@ scheduled on the node. Existing pods on the node are not affected.
 +
 [source,bash]
 ----
-$ oc adm manage-node <node1> <node2> --schedulable=false
+$ oc adm cordon <node>
```
Comment (mburke5678) [inline modules/nodes-nodes-working-marking.adoc]: @sjenning @smarterclayton It appears through my *arduous* testing that you can only specify one node with the `oc adm cordon` command. Please let me know if this is not the case.

**Comment by mburke5678** [comment]
@sjenning @smarterclayton PTAL

File: modules/nodes-nodes-working-marking.adoc (hunk 1 above)
Comment (sjenning) [reply modules/nodes-nodes-working-marking.adoc]: this appears to be true
```
oc adm cordon node1 node2
error: USAGE: cordon NODE [flags]
See 'oc adm cordon -h' for help and examples.
```
label selector method seems to be busted
```
$ oc adm cordon node --selector=beta.kubernetes.io/arch=amd64
error: cannot specify both a node name and a --selector option
See 'oc adm cordon -h' for help and examples.
```

**Comment by sjenning** [comment]
LGTM

**Pull request merged**